css-layout-classes/style.css

```diff
@@ -0,0 +1,75 @@
+/* font default for entire webpage */
+body {
+  font-family: sans-serif;
+  background: #ddd;
+}
+h1 {
+  text-align: center;
+  font-weight: bold;
+}
+
+h2 {
+  text-align: center;
+  font-size: 20px;
+  font-weight: bolder;
+}
+
+/* flexing the whole flexbox container div */
+#flexbox-container {
+  display: flex;
+  justify-content: center;
+}
+
+#flexbox-container > .flex-para {
+  min-width: 250px;
+  max-width: 450px;
+}
+
+.flex-para-one {
+  min-width: 250px;
+  max-width: 180px;
+}
+
+/* 2nd row column flex */
+
+.row {
+  flex-basis: 100%;
```

Comment:
`flex-basis` is a child property which means it only works on elements who's direct parent have `display: flex`